tests/integration/gitlab/test_service.py

```diff
@@ -87,3 +89,20 @@ def test_service_without_auth(self):
         service = GitlabService()
         assert service.gitlab_instance
         assert service.get_project(namespace="gnachman", repo="iterm2").exists()
+
+    def test_list_projects_with_namespace_input(self):
+        service = GitlabService(os.environ["GITLAB_TOKEN"])
+        name_of_the_repo = "inkscape"
+        number_of_projects = 10
+        projects = service.list_projects(namespace=name_of_the_repo)
```

Comment:
```suggestion
        projects = self.service.list_projects(namespace=name_of_the_repo)
```

the same here as well :)